### DISCUSSION TITLE
Is it ok for package.json "files" field to not include `syntaxes/` and `language-configuration.json`?

### DISCUSSION THREAD

**Discussion by trusktr** [eclipse-langium/langium#920] [Q&A] [answered]
I see the `files` field does not include everything that makes up the package. Is this ok?

**Comment by msujew** [comment, marked as answer]
Hey @trusktr,

the `files` entries of the `package.json` are only for npm to know what to package & upload when performing an `npm publish`. Our default yeoman generator does not generate anything that's supposed to be uploaded as an npm package, but rather a vscode extension. vscode manages it's own included (or rather ignored) files using the [`.vscodeignore`](https://github.com/langium/langium/blob/main/packages/langium-vscode/.vscodeignore) file. In this context, it is pretty much irrelevant what the `files` entry contains.
